Docs/source/usage/parameters.rst

```diff
@@ -504,6 +504,10 @@ Particle initialization
     number of particles per theta is at least two times the number of azimuthal modes requested.
     (It is recommended to do a convergence scan of the number of particles per theta)
 
+* ``<species_name>.random_theta`` (`bool`) optional (default `0`)
```

Comment:
If the new default is `true`, I think we should also update the documentation accordingly:
```suggestion
* ``<species_name>.random_theta`` (`bool`) optional (default `1`)
```